ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -153,6 +156,18 @@ def backup(host):
         raise AssertionError('Backup directory not found in output')
 
 
+def user_add(host, username):
+    arg = ['ipa', 'user-add', '--first', 'None', '--last', 'None', username]
```

Comment:
I think this one should be named as `args`, but it's just an opinion of mine.

Reply:
I believe we could also extend the `Host` class with a `run_ipa_command`. These two functions here (`user_add` and `user_show`) are copy-pasta in the first place, the only thing that changes are the arguments, which apparently calls for a generalization around them rather than having a separate function for each IPA CLI command.